### PR TITLE
fix(ojoi-application): Fixing bugs from testing

### PR DESCRIPTION
**Pull request by jonbjarnio**
# Bug fixes

## What
- Fixing bugs found during testing

## Why
- For better user experience 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by Toti91 — APPROVED** [review]
ui and application files LGTM

**Review by busla — APPROVED** [review]
infra LGTM

**Review by Valur — APPROVED** [review]

File: libs/api/domains/ministry-of-justice/src/lib/ministryOfJustice.resolver.ts [hunk 1]
```diff
@@ -31,42 +30,37 @@ export class MinistryOfJusticeResolver {
   @Query(() => AdvertResponse, {
     name: 'ministryOfJusticeAdvert',
   })
-  @Audit()
```
Comment (saevarma) [inline libs/api/domains/ministry-of-justice/src/lib/ministryOfJustice.resolver.ts:34]: Is it no longer desired to audit log these requests on our side?

Comment (Valur) [reply libs/api/domains/ministry-of-justice/src/lib/ministryOfJustice.resolver.ts:34]: I need to be able to fetch this data in the web application, These audit lines were causing the requests to return 401 Unauthorized

Comment (saevarma) [reply libs/api/domains/ministry-of-justice/src/lib/ministryOfJustice.resolver.ts:34]: Ahh I see, then its all good.

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**